app/components/shared/Autocomplete/Dialog/index.js

```diff
@@ -100,7 +100,7 @@ class AutocompleteDialog extends React.PureComponent {
     }
 
     return (
-      <div className="block" style={{ width: "100%", minHeight: 300 }}>
+      <div className="block" style={{ width: "100%", maxHeight: 250, height: '50vh', overflow: 'auto' }}>
```

Comment:
I tested it, it's not too bad! Any suggestions for making it less crazy on desktop, when you have a larger screen and dozens of users in the org? (It's tedious to set that data up for dev, but it appears we never designed for those cases)

There's actually lots of other teams stuff that's more broken on mobile too! But I didn't get time to fix much of it 😞